locales/en.yml

```diff
@@ -283,3 +286,16 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+popup:
+  auth:
+    title: Members only
+    body: Sign in to {action}
+    signin: Sign in
+TButton:
+  login: Login to {action}
+TDropdown:
+  title: report or share
+TReaction:
+  label:
```

Comment:
If it's the only one property, the dissolve it and move children one level up. 

P.S. Otherwise I would recommend to use `TReaction.star.label` as it's label that belongs to the star, and not star that belongs to the label. See my point?

Reply:
yes